pandas/plotting/_core.py

```diff
@@ -625,11 +626,10 @@ def _get_plot_backend():
     The backend is imported lazily, as matplotlib is a soft dependency, and
     pandas can be used without it being installed.
     """
-    try:
-        import pandas.plotting._matplotlib as plot_backend
-    except ImportError:
-        raise ImportError("matplotlib is required for plotting.")
-    return plot_backend
+    backend_str = pandas.get_option('plotting.backend')
```

Comment:
Is this still necessary? Or is it imported by the validation of the cb? not sure if called the first time.

Reply:
Good point. It should be imported in the validation of the cb (and loaded and added to `sys.modules`). But I think we still need this, because we need to obtain a handler of the module. So, this won't reload it, since Python caches modules, but will return us the module object, so we can access its attributes, call its functions...